akka-http-core/src/main/scala/akka/http/scaladsl/model/HttpEntity.scala

```diff
@@ -72,6 +72,9 @@ sealed trait HttpEntity extends jm.HttpEntity {
       .via(new akka.http.impl.util.ToStrict(timeout, contentType))
       .runWith(Sink.head)
 
+  def discardEntityBytes(mat: Materializer): HttpMessage.DiscardedEntity =
```

Comment:
I agree with you. I consider them suitable and descriptive names. Another argument is that
change discardEntityBytes() to discardBytes() on HttpMessage will break at least 32 places
and I am not sure if is out of the scope of this PR.